### PR TITLE
move validate release to hugegraph-doc

### PR DESCRIPTION
**Pull request by z7658329**
test see:
https://github.com/z7658329/hugegraph/actions/runs/4150859892

### CODE REVIEW

**Review by javeme — DISMISSED** [review]

**Review by javeme — APPROVED** [review]

File: .github/workflows/validate-release.yml
Comment (javeme) [inline .github/workflows/validate-release.yml:2]: can we use the remote script and delete this file?

Comment (javeme) [inline .github/workflows/validate-release.yml:2]: can we just delete this file?

File: hugegraph-dist/scripts/validate-release.sh
Comment (javeme) [inline hugegraph-dist/scripts/validate-release.sh:151]: "name:$i" -- prefer `name $i`

Comment (javeme) [inline hugegraph-dist/scripts/validate-release.sh:151]: "name:$i" -- prefer` name $i`

File: hugegraph-dist/scripts/validate-release.sh
Comment (javeme) [inline hugegraph-dist/scripts/validate-release.sh:28]: also move this file to hugegraph-doc repo?

File: .github/workflows/validate-release.yml
Comment (z7658329) [reply .github/workflows/validate-release.yml:2]: > can we use the remote script and delete this file?

ok, we can see the apache release validate ci in hugegraph-doc

File: hugegraph-dist/scripts/validate-release.sh
Comment (z7658329) [reply hugegraph-dist/scripts/validate-release.sh:28]: > also move this file to hugegraph-doc repo?

It can be discussed that the server keep validate shell script while hugegraph-doc keep ci actions ?
@javeme @imbajin

File: hugegraph-dist/scripts/validate-release.sh
Comment (z7658329) [reply hugegraph-dist/scripts/validate-release.sh:151]: > "name:$i" -- prefer `name $i`

ok

File: hugegraph-dist/scripts/validate-release.sh
Comment (imbajin) [reply hugegraph-dist/scripts/validate-release.sh:28]: > > also move this file to hugegraph-doc repo?
> 
> It can be discussed that the server keep validate shell script while hugegraph-doc keep ci actions ? @javeme @imbajin

Both fine, and wait until the **first release** is completed. After confirming the stability, we can merge current PR, 
could leave it alone for now

**Review by javeme — DISMISSED** [review]

**Review by javeme — APPROVED** [review]

File: hugegraph-dist/pom.xml
Comment (imbajin) [inline hugegraph-dist/pom.xml:184]: **TODO:** use curl to replace it (and refer a better way to handle it -- like maven property)

**Review by imbajin — APPROVED** [review]

**Comment by imbajin** [comment]
@z7658329 Maybe we shall merge the PR in doc first?

**Pull request merged**
